aml-development/frontend
# feat(listing): updated listing page

AMLOS-557

**Description**
As a user I want to be able to open a Listing to view details about the listing.

**Acceptance Criteria**
Listing view supports...
 Listing Quick View Overview Tab
 Listing Quick View Reviews Tab
 Listing Quick View Details Tab
 Listing Quick View Resources Tab
 Listing Quick View Admin Tab

**How to Test**
Pull `updated_listing_page` from `frontend`
Ensure that going to the listing page loads all the sections of a listing appropriately.  For this PR, we additionally have the ability to write a review in the reviews section and bookmark a listing and launch the listing page from the header section.
